components/resources/ResourceOccurrences.js

```diff
@@ -54,7 +54,9 @@ const ResourceOccurrences = (props) => {
             )}
           </>
         ) : (
-          <p className={styles.notFound}>No resource found for {resourceUri}</p>
+          <p className={styles.notFound}>
+            No resource found for &quot;{resourceUri}&quot;
```

Comment:
Six of one, half a dozen of the other

```suggestion
            No resource found for {`"${resourceUri}"`}
```